tests/test_units.py

```diff
@@ -253,12 +253,12 @@ def test_amount2lwethickness(snw_series):
 
     swe = amount2lwethickness(snw, out_units="mm")
     # FIXME: Asserting these statements shows that they are not equal
```

Comment:
```suggestion
```
I guess this seems fixed ?

Reply:
Ah well it isn't haha.

Reply:
@coxipi let me know if you can tackle this soon! Thanks!